indra/llcommon/llapp.cpp

```diff
@@ -414,6 +414,22 @@ void LLApp::writeMiniDump()
 {
 }
 
+// static
+void LLApp::quit(bool force)
+{
+    if (LLApp* app = instance())
+    {
+        if (force)
+        {
+            app->forceQuit(); // Puts the viewer into 'shutting down without error' mode.
+        }
+        else
+        {
+            app->requestQuit(); // Request a quit. A kinder, gentler quit.
+        }
+    }
```

Comment:
I think we can assume that any caller of `LLApp::quit()` might behave badly if the call were to return. What if `instance()` returns `nullptr`? I think that case should be an `LL_ERRS()` hard stop, or at least a `std::exit()` call.

Reply:
Done